src/ck-core/ckrdma.C

```diff
@@ -854,11 +854,10 @@ int getSrcPe(envelope *env) {
   PUP::fromMem up((void *)((CkMarshallMsg *)EnvToUsr(env))->msgBuf);
   up|numops;
```

Comment:
PUPing numops is useless here and below in getRootNode(). That can be done here or in a separate patch

Reply:
Isn't the goal of this just to unpack in the same way it was packed before?

Reply:
Unpacking numops and rootNode is required since that was how it was packed.